### ISSUE TITLE
Planning Accessible Audio and Video Media

### ISSUE THREAD

**Issue by Sylduch-Conseil** [w3c/wai-media-guide#77]
In the checklist, first bullet, clarify A.S.:
"If there are no “A”s, then it is not required in WCAG."
While writing this issues, I discovered you may talk about a aa, or aaa levels.
Suggestion: if nothing is specified, level A, AA, or AAA, then it is not required in WCAG.

**Comment by shawna-slh** [comment]
I'd rather not repeat A, AA, AAA. I edited it to:

> The checklists below... include:
> * What is required in the Web Content Accessibility Guidelines (WCAG) standard at Level A, AA, and AAA. _([WCAG](https://wai-media-guide.netlify.com/design-develop/media/planning/#wcag-standard) is explained below.)_
> * What is needed to meet user needs, beyond WCAG. (If it doesn't have any 'A', then it is not required in WCAG.)

I hope that is more clear. (The 'A' has single quotes marks around it.)
